tests/market.test.ts

```diff
@@ -349,7 +350,10 @@ describe("Market", () => {
 
 		// Should get expiry after back bet
 		const expiry = await market.getExpiry(0);
-		expect(expiry).to.equal(end + 2592000, "Should have expiry set");
+		expect(expiry).to.equal(
```

Comment:
could add an additional step to fast forward to when the timeout unlocks and attempt to interact

Reply:
I think that test exists.

Reply:
https://github.com/horse-link/contracts.horse.link/blob/27027d6d9bba2068e8f7e64cc8d053da00ae8993/tests/market.test.ts#L683